### PR TITLE
Update example-projects.md

### PR DESCRIPTION
**Pull request by gmondok**
Added tutorials section.

### CODE REVIEW

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Content looks great! Can you use markdown links instead of `a` tags?

**Comment by gmondok** [comment]
> Content looks great! Can you use markdown links instead of `a` tags?

I had replicated the same link format that the doc was already using. Do you want me to change all the old links over to markdown links too?

**Comment by StephenFluin** [comment]
Sure that would be great. We're trying to slowly move the site to more pure
markdown as we touch it more and more.

File: docs/Additional Information/example-projects.md [hunk 1]
```diff
@@ -9,6 +9,20 @@ hidden: false
...
 
+## Hackathon Tutorials
+
+Further exploration into how some of the hackathon winners developed their entries.
+
+|Name|Description|Tutorial|
+|:---|:---|:---|
+|**Link My Ride**|A decentralized Tesla rental platform.|<a href="https://blog.chain.link/create-tesla-smart-contract-rental/" target="_blank" rel="noreferrer, noopener">Go</a>|
+|**Open Library Project**|A library platform for borrowing books on-chain.|<a href="https://blog.chain.link/rfid-blockchain-integration-with-chainlink-external-adapters/" target="_blank" rel="noreferrer, noopener">Go</a>|
+|**Marine Insurance**|Parametric insurance for the marine industry.|<a href="hhttps://blog.chain.link/build-a-marine-insurance-smart-contract-with-chainlink/" target="_blank" rel="noreferrer, noopener">Go</a>|
```
Comment (StephenFluin) [inline docs/Additional Information/example-projects.md:20]: Typo here in URL as well: 'hhttps'

**Comment by StephenFluin** [comment]
Can you squash these two commits into one?

```
git reset HEAD~1
git commit -a --amend
git push -f
```

**Comment by gmondok** [comment]
> Can you squash these two commits into one?
> 
> ```
> git reset HEAD~1
> git commit -a --amend
> git push -f
> ```

Done.

**Comment by StephenFluin** [comment]
Did the git push -f succeed to your branch?  Not seeing any updates to the PR.

Sorry about the extra steps. We are just trying to keep each PR to one commit to have a nice clean history.

**Pull request closed without merging**